### PR TITLE
Dt 883 splash screen

### PR DESCRIPTION
**Pull request by siren**
implements 
DT-883
DT-884
DT-886
DT-887
DT-890

### CODE REVIEW

**Comment by hannesj** [comment]
The map image should probably be defined in the style

File: app/routes.js [hunk 1]
```diff
@@ -15,13 +15,16 @@ import LoadingPage from './page/loading';
 import Error404 from './page/404';
 import StyleGuidelinesPage from './page/style-guidelines';
 import AddFavouritePage from './page/add-favourite';
+import SplashOrComponent from './component/splash/splash-or-component';
 
 import TopLevel from './component/top-level';
 import { StopQueries, TripQueries, RouteQueries } from './queries';
 
+let IndexOrSplash = ()=><SplashOrComponent><IndexPage/></SplashOrComponent>
```
Comment (hannesj) [inline app/routes.js]: not used?

File: app/translations.coffee [hunk 2]
```diff
@@ -259,6 +261,8 @@ translations =
     'search': 'Sök'
     'in-addition': 'Även'
     'choose-stop': 'Välj hållplats'
+    'loading': 'loading'
```
Comment (hannesj) [inline app/translations.coffee]: Laddar

File: app/translations.coffee [hunk 3]
```diff
@@ -259,6 +261,8 @@ translations =
...
+    'skip-positioning': 'skip-positioning'
```
Comment (hannesj) [inline app/translations.coffee]: Hoppa över

**Comment by samuliheljo** [comment]
Splash screen:
- English text "Give origin" -> "Set origin"

When search dialog opens, it should not say "Search" but "Origin" and "Lähtöpaikka"

File: app/config.default.coffee [hunk 4]
```diff
@@ -146,4 +146,8 @@ module.exports =
     accessibility:
       available: true
   areaPolygon: [[ 18.776, 60.3316 ], [ 18.9625, 60.7385 ], [ 19.8615, 60.8957 ], [ 20.4145, 61.1942 ], [ 20.4349, 61.9592 ], [ 19.7853, 63.2157 ], [ 20.4727, 63.6319 ], [ 21.6353, 63.8559 ], [ 23.4626, 64.7794 ], [ 23.7244, 65.3008 ], [ 23.6873, 65.8569 ], [ 23.2069, 66.2701 ], [ 23.4627, 66.8344 ], [ 22.9291, 67.4662 ], [ 23.0459, 67.9229 ], [ 20.5459, 68.7605 ], [ 20.0996, 69.14 ], [ 21.426, 69.4835 ], [ 21.9928, 69.4009 ], [ 22.9226, 68.8678 ], [ 23.8108, 69.0145 ], [ 24.6903, 68.8614 ], [ 25.2262, 69.0596 ], [ 25.4029, 69.7235 ], [ 26.066, 70.0559 ], [ 28.2123, 70.2496 ], [ 29.5813, 69.7854 ], [ 29.8467, 69.49 ], [ 28.9502, 68.515 ], [ 30.4855, 67.6952 ], [ 29.4962, 66.9232 ], [ 30.5219, 65.8728 ], [ 30.1543, 64.9646 ], [ 30.9641, 64.1321 ], [ 30.572, 63.7098 ], [ 31.5491, 63.3309 ], [ 31.9773, 62.9304 ], [ 31.576, 62.426 ], [ 27.739, 60.1117 ], [ 26.0945, 59.8015 ], [ 22.4235, 59.3342 ], [ 20.2983, 59.2763 ], [ 19.3719, 59.6858 ], [ 18.7454, 60.1305 ], [ 18.776, 60.3316 ]]
-  defaultPosition: [60.317429, 24.9690395]
+  #default origin endpoint to use when user is autside of area
+  defaultEndpoint:
+    address: "Lentoasema"
```
Comment (hannesj) [inline app/config.default.coffee]: Helsinki-Vantaan Lentoasema?

**Comment by samuliheljo** [comment]
img_0796

Mobile lumia weirdnes

**Comment by hannesj** [comment]
Move OriginPopup inside OriginMarker and PositionMarker in the future?

**Pull request merged**